docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,266 @@
+	
+- `errorOverlayUrl` is a URL to the overlay image to show in place of the overlay that failed to load.
+
+- `alt` sets the HTML [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt) attribute to provide an alternative text description of the image. Alternative text is essential information for screen reader users. It can also benefit people during poor network connectivity, in the case the image fails to load. Moreover, it can improve the SEO of a website.
+
+- `interactive` is `false` by default. If `true`, the image overlay will emit mouse events when clicked or hovered.
+
+You can find other options of `L.ImageOverlay` in the [documentation](/reference.html#imageoverlay).
+
+{% include frame.html url="example-image.html" %}
+
+### `VideoOverlay`
+
+Video used to be a hard task when building a webpage, until the [`<video>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) was made available.
+
+Nowadays, we can use the following HTML code:
+
+```
+<video width="500" controls>
+	<source src="https://www.mapbox.com/bites/00188/patricia_nasa.webm" type="video/webm">
+	<source src="https://www.mapbox.com/bites/00188/patricia_nasa.mp4" type="video/mp4">
+</video>
+```
+
+To display this video:
+
+<video width="500" controls>
+<source src="https://www.mapbox.com/bites/00188/patricia_nasa.webm" type="video/webm">
+<source src="https://www.mapbox.com/bites/00188/patricia_nasa.mp4" type="video/mp4">
+</video>
+
+If a video can be shown in a webpage in this way, then Leaflet can display it inside a map. It is important that the videos are prepared in such a way that they will fit the map: The video should have a "north-up" orientation, and its proportions should fit the map. If not, it will look out of place.
+
+#### Creating a map
+
+First of all, create a Leaflet map and add a background `L.TileLayer` in the usual way:
+
+```
+var map = L.map('map').setView([37.8, -96], 4);
+
+L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=' + mapboxAccessToken, {
+	id: 'mapbox/satellite-v9',
+	attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
+		'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+	tileSize: 512,
+	zoomOffset: -1
+}).addTo(map);
+```
+
+Then, we'll define the geographical bounds that the video will cover. This is an instance of [`L.LatLngBounds`](/reference.html#latlngbounds), which is a rectangular shape:
+
+```
+var latLngBounds = L.latLngBounds([[32, -130], [13, -100]]);
+```
+If you want to see the area covered by a `LatLngBounds`, use a [`L.Rectangle`](/reference.html#rectangle):
+```
+L.rectangle(latLngBounds).addTo(map);
+
+map.fitBounds(latLngBounds);
+```
+
+{% include frame.html url="example-bounds.html" %}
```

Comment:
This is now mentioned under ImageOverlay as well. I wonder if this should be removed, @Falke-Design?

Reply:
Or should it be stated a 3rd time under SVGOverlay?

Reply:
If it is mentioned twice, then remove this one

Reply:
removed